### DISCUSSION TITLE
argocd doesn't show all fields from the values.yaml (for example the comments)

### DISCUSSION THREAD

**Discussion by 1mfaasj** [argoproj/argo-cd#8066] [Q&A]
Hi,
Ive added a helm repository and I'm trying to create a new application from it.
```
For instance: https://nextcloud.github.io/helm
Chart: nextcloud
```
I can choose the values.yaml file from the chart or edit it manually in this screen:
![image](https://user-images.githubusercontent.com/71582854/147736581-d8425b7f-0105-4048-a0a7-86039808bf8c.png)
but it seems it doesn't show ALL values from the values.yaml in Argocd.
when I view the actual values.yaml from the chart: https://github.com/nextcloud/helm/blob/master/charts/nextcloud/values.yaml
I see that there are even more values in it, such as a number of 'comments' with a # in front of it.
Is there a way to view these values in Argocd?
![image](https://user-images.githubusercontent.com/71582854/147736690-92ebeb29-5452-4f96-98c6-ffb163c50b60.png)

in the comments like in the last sceenshot there are comments in the values.yaml but important ones that I definitely want to see in Argocd.

**Comment by LiberoScarcelli** [comment]
Hi,

I am having the same issue here but these parameters are not commented out

![helm](https://user-images.githubusercontent.com/101608253/179473246-fba74ebb-9824-40d4-9a69-9135e9f329dc.jpg)

![argocd](https://user-images.githubusercontent.com/101608253/179473275-e3f5b14e-c8f4-490b-a05f-f6b879f7af0f.jpg)

**Reply by crenshaw-dev** [reply]
Looks like a bug related to object-type params. I see you opened an issue here: https://github.com/argoproj/argo-cd/issues/10020

**Comment by crenshaw-dev** [comment]
Can you open an enhancement request?

Supporting commented-out fields might be tricky, because we'd have to detect commented YAML vs. plain old comments.
